windows-vm-configure-winrm/README.md

```diff
@@ -0,0 +1,32 @@
+# Deploy a Windows VM and Configures a WinRM Https
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fwindows-vm-configure-winrm%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+
+
+Description of Template
+=======================
+This template allows you to deploy a simple Windows VM using a few different options for the Windows version. 
+This will then configure a WinRM https listener by creating a new test certificate.
+
+The template uses a custom script extension which executes the script 'https://raw.githubusercontent.com/pavanadepu2/MyRepo/master/ConfigureWinRM.ps1' on the target machine.
+This script creates a self signed certificate and configures the WinRM Https listener using the certificate's thumbprint.
+
+
+
+How to connect to a Target Azure VM post WinRM configuration
+============================================================
+Use the below script to connect to an azure vm post winrm configuration. Assign the exact fqdn of your azure vm to $hostname.
+The script pops up a credential window, provide the credentials of azure vm.
+
+	$hostName="mywindowsvm.westus.cloudapp.azure.com"
```

Comment:
After reproing this locally, the hostname is actually going to be of type <dnsLabelPrefix>.<resourceGroupLocation>.cloudapp.azure.com otherwise somebody could get confused and paste this directly in